service/controller/key/key.go

```diff
@@ -112,17 +111,13 @@ const (
 
 const (
 	KubernetesNetworkSetupDocker = "0.2.0"
-	kubernetesAPIHealthzVersion  = "0.1.0"
+	kubernetesAPIHealthzVersion  = "0.1.1"
```

Comment:
These versions strings are so hard to maintain. Not sure how to improve them though.

Reply:
I agree. We will get rid of them in the coming months I hope.